staging/src/k8s.io/apiserver/pkg/endpoints/handlers/get.go

```diff
@@ -81,22 +81,14 @@ func getResourceHandler(scope *RequestScope, getter getterFunc) http.HandlerFunc
 }
 
 // GetResource returns a function that handles retrieving a single resource from a rest.Storage object.
-func GetResource(r rest.Getter, e rest.Exporter, scope *RequestScope) http.HandlerFunc {
+func GetResource(r rest.Getter, scope *RequestScope) http.HandlerFunc {
 	return getResourceHandler(scope,
 		func(ctx context.Context, name string, req *http.Request, trace *utiltrace.Trace) (runtime.Object, error) {
 			// check for export
 			options := metav1.GetOptions{}
 			if values := req.URL.Query(); len(values) > 0 {
-				exports := metav1.ExportOptions{}
-				if err := metainternalversionscheme.ParameterCodec.DecodeParameters(values, scope.MetaGroupVersion, &exports); err != nil {
-					err = errors.NewBadRequest(err.Error())
-					return nil, err
-				}
-				if exports.Export {
-					if e == nil {
-						return nil, errors.NewBadRequest(fmt.Sprintf("export of %q is not supported", scope.Resource.Resource))
-					}
-					return e.Export(ctx, name, exports)
+				if len(values.Get("export")) > 0 {
```

Comment:
ignore anything that would have resulted in exports.Export == false, right? (e.g. case-insensitive `false`, `0`, `""`)

Reply:
> ignore anything that would have resulted in exports.Export == false, right? (e.g. case-insensitive `false`, `0`, `""`, per `Convert_Slice_string_To_bool`)

sure